main/Helper.cpp

```diff
@@ -37,6 +37,14 @@
 #include <time.h>
 #endif
 
+#if defined(__FreeBSD__) 
+// Check if NetBSD / OpenBSD or DragonFly need that at well?
+#include <pthread_np.h>
+#ifndef PTHREAD_MAX_MAMELEN_NP
+#define PTHREAD_MAX_NAMELEN_NP 10 	// Arbitrary
```

Comment:
I think this is a bit to small. Why not make it 32 like on other systems ?

Reply:
Please do not include <pthread_np.h> on NetBSD, it does not exist and it will break the build.